src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/jvm/ThreadList.java

```diff
@@ -237,16 +241,13 @@ private static void parseLine(String line) {
         nameMap.put(t.threadName, t); //XXX: we assume no collisions
     }
 
-    private static void createMap(InputStream in) {
-        try (BufferedReader br = new BufferedReader(new InputStreamReader(in));) {
-            String line = null;
-            while ((line = br.readLine()) != null) {
-                if (line.contains("tid=")) {
-                    parseLine(line);
-                }
+    private static void createMap(InputStream in) throws Exception {
```

Comment:
throws IOException?

Reply:
Kept it generic; as we handling all exceptions on the caller side